libs/sdk-ui-dashboard/src/presentation/drill/utils/drillDownUtils.ts

```diff
@@ -16,3 +18,30 @@ export function getDrillDownAttributeTitle(drill: IDrillDownDefinition, drillEve
         )
         .map((intersectionAttributeItem) => intersectionAttributeItem.attributeHeaderItem.name)[0];
 }
+
+/**
+ * Implicit drillDown has lower priority, so needs to be removed when other drill config exists for the same attribute
+ *
+ * @internal
+ */
+export function filterDrillFromAttributeByPriority(
+    drillDefinitions: DashboardDrillDefinition[],
+): DashboardDrillDefinition[] {
+    const nonImplicitDrillDown = drillDefinitions.filter(
+        (dd) => !isDrillDownDefinition(dd) && !isDrillToLegacyDashboard(dd),
+    );
+
+    const nonImplicitDrillDownIds = nonImplicitDrillDown.map((d) => {
+        if (!isDrillToLegacyDashboard(d)) {
```

Comment:
Will this if ever be false? These will be filtered out by the filter on line 30 no?

Reply:
Discussed offline, it is because of typescript inference